tests/stress/enc/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+set(EDL_FILE ../stress.edl)
+
+add_custom_command(
+  OUTPUT stress_t.h stress_t.c
+  DEPENDS ${EDL_FILE} edger8r
+  COMMAND edger8r --trusted ${EDL_FILE} --search-path
```

Comment:
Given that #2961 has been merged, can you please check that this test is not missing any necessary EDL imports from system or platform?

This comment also applies to the host/CMakeLists.txt

Reply:
add necessary EDL imports to all CMakeLists.txt